packages/playwright/src/runner/runner.ts

```diff
@@ -134,4 +140,82 @@ export class Runner {
     ]);
     return { status };
   }
+
+  async printWarnings(lastRun: LastRunReporter) {
```

Comment:
I'm not really sure this is the right place for this to live. This is where the other test parsing paths live, so I put it here for now, but I'm open to suggestions on where to move it.